testing/test_test.py

```diff
@@ -0,0 +1,99 @@
+from asyncio import format_helpers
+import random
+import pytest
+import numpy as np
+
+from batlib.model import Model
+from batlib.model import BatteryCoefficients
+
+
+def voltage_noise_array(final_value, noise_range, noise_voltage, length):
+    """
+    Creates an array of random voltages set around a given range of noise 
+    at a voltage value.
+    After half of the length of the array is reached, the array will start
+    to settle around a final voltage value.
+
+    :param final_value:     final voltage value to settle to
+    :param noise_range:     added noise range to a noise voltage
+    :param noise_voltage:   voltage where noise is generated around
+    :param length:          length of the data array 
+    :return:                generated voltage array
+    """
+    volt_data = [0 for i in range(0, length-1)]
+    volt_data = [(random.uniform(-noise_range, noise_range)+noise_voltage)
+                 for i in range(0, int(length/2)-1)]
+    for i in range(int(length/2), length-1):
+        volt_data.append(final_value)
+    return volt_data
+
+
+def test_model_0():
+    """
+    This is a test for the functions in the model.py class.
+    The voltage data is being initialized with a final settling 
+    voltage value of 12.3, noise range of .5, noise voltage value 
+    of 11.9 and length of 100
+    """
+    struct_1 = BatteryCoefficients()
+    model_1 = Model(struct_1)
+    volt_data = voltage_noise_array(12.3, 0.5, 11.9, 100)
+    for i in range(0, len(volt_data)-1):
+        model_1.setInput(volt_data[i], 1e3, 25)
+        print(model_1.getCapacity())
+        model_1.update(0.25 * 1e6)
```

Comment:
dt show be in seconds - make sure that all the update loop will work in seconds, not microseconds